plugins/superio/fu-superio-device.c

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fcntl.h>
+#include <string.h>
+#include <sys/errno.h>
+
+#include <glib/gstdio.h>
+
+#include "fu-superio-device.h"
+
+#define FU_PLUGIN_SUPERIO_TIMEOUT	5 /* s */
+
+#define KB_IBF			(1 << 1)  /* i/p buffer full */
```

Comment:
I think I've seen this in a few places now in fwupd.  Maybe we should have a generic `BIT()` macro like the kernel has.

Reply:
This is the style used in other projects like flashrom -- I think i'd rather keep the bitshift.